File: DB-Advanced-C#-EntityFramework-Core/Version1/OOP_Intro/RawData/Cargo.cs
namespace RawData
{
    class Cargo
    {
        private int weight;
        private string cargoType;

        public Cargo() { }
        public Cargo(int weight, string cargoType)
        {
            this.Weight = weight;
            this.CargoType = cargoType;
        }

        public string CargoType
        {
            get { return cargoType; }
            set { cargoType = value; }
        }

        public int Weight
        {
            get { return weight; }
            set { weight = value; }
        }
    }
}

File: Java-OOP-Basics/Problems Course Instance 2016_2017/src/Lesson01DefiningClasses/Exercise/pr06_raw_data/model/Cargo.java
package Lesson01DefiningClasses.Exercise.pr06_raw_data.model;

public class Cargo {
    private int weight;
    private String type;

    Cargo(String type, int weight) {
        this.weight = weight;
        this.type = type;
    }

    public String getType() {
        return type;
    }
}
